### PR TITLE
1 add project config setup

### PR DESCRIPTION
**Pull request by PeterHamfelt**

### CODE REVIEW

File: tests/checkers/test_pandas/test_import_pandas.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+        with self.assertAddsMessages(
+            pylint.testutils.MessageTest(
+                msg_id="pandas-importfrom",
+                confidence=HIGH,
+                node=pandas_importfrom_node,
+            ),
+            ignore_position=True,
+        ):
+            self.checker.visit_importfrom(pandas_importfrom_node)
```
Comment (Pierre-Sassoulas) [inline tests/checkers/test_pandas/test_import_pandas.py:53]: This make me want to make our functional test more generic so its easily usable in plugins' tests.

**Review by Pierre-Sassoulas — APPROVED** [review]

File: CONTRIBUTORS.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Maintainers
```
Comment (Pierre-Sassoulas) [inline CONTRIBUTORS.txt:1]: This file is a mess to maintain, I would use the git history and github autorisation as sources of truth. (Pylint has something similar for historical reasons only)

Comment (PeterHamfelt) [reply CONTRIBUTORS.txt:1]: True indeed, I will have a look at Pylint and fix this.

File: tests/checkers/test_pandas/test_import_pandas.py (hunk 1 above)
Comment (PeterHamfelt) [reply tests/checkers/test_pandas/test_import_pandas.py:53]: Okay, can you recommend a test case that aligns with best practices?

File: .gitignore [hunk 3]
```diff
@@ -157,4 +157,12 @@ cython_debug/
...
-#.idea/
+.idea/
+
+# Mr Developer
+.mr.developer.cfg
+.project
+.pydevproject
+.idea
+env.txt
+poetry.lock
```
[the remark this answers is not in the document]
Comment (PeterHamfelt) [reply .gitignore:168]: Great, I will add a global .gitignore in the upcoming PR.

File: tests/checkers/test_pandas/test_import_pandas.py (hunk 1 above)
Comment (Pierre-Sassoulas) [reply tests/checkers/test_pandas/test_import_pandas.py:53]: Just a passing command, nothing to do from your end, but functional is documented here: https://pylint.readthedocs.io/en/stable/development_guide/contributor_guide/tests/writing_test.html#functional-tests (for pylint, how to use in a plugin is not documented, possibly not trivial).

File: CONTRIBUTORS.txt (hunk 2 above)
Comment (Pierre-Sassoulas) [reply CONTRIBUTORS.txt:1]: I suggested having nothing, as maintaining a file is a lot of work. (Had to do tooling for pylint, but it's not ideal).

File: .gitignore (hunk 3 above)
[the remark this answers is not in the document]
Comment (Pierre-Sassoulas) [reply .gitignore:168]: You need to add it locally so you don't have to add env specific ignore to all your projects (``git config --global core.excludesfile ~/.global_gitignore`` then edit ``global_gitignore`` in your home dir.

**Pull request merged**